lua/kickstart/plugins/autoformat.lua

```diff
@@ -0,0 +1,74 @@
+-- autoformat.lua
+--
+-- Use your language server to automatically format your code on save.
+-- Adds additional commands as well to manage the behavior
+
+return {
+  'neovim/nvim-lspconfig',
```

Comment:
TODO: For myself, could add null-ls here since I know that's a big hang up for a lot of people

Reply:
meh, i think i'll do this later if I want it